launcher/spec/launch_policy.go

```diff
@@ -54,6 +85,16 @@ func (p LaunchPolicy) Verify(ls LaunchSpec) error {
 		return fmt.Errorf("CMD is not allowed to be overridden on this image")
 	}
 
+	if p.AllowedLogRedirect == never && ls.LogRedirect {
+		return fmt.Errorf("log is not allowed to be redirected on this image")
```

Comment:
"logging redirection not allowed by image"

Reply:
done